docs/components/Chips/Chips.stories.mdx

```diff
@@ -144,3 +144,34 @@ colors for parity with Jobber's mobile platform, but we will not be implementing
 this is for the web components as of this first iteration of our web Chip. The
 mobile use-case is very specific to a filtering use case that we as of now do
 not anticipate will be required for the web.
+
+### Chips Composition with Standalone Chip (Web)
+
+Standalone Chip's `role`, `<Chip.Prefix>`, `<Chip.Suffix>` `heading` and
```

Comment:
The point of this section feels a bit fuzzy, it contains

- a list of features that don't work
- an example of how to work around one of those features
- an example of how not to use one of the features we've already told you won't work

Could this be a clearer "Do/Don't"? We have examples in our [frontend style guide](https://atlantis.getjobber.com/?path=/docs/guides-frontend-styleguide--docs#spacing-margins-and-padding).

Reply:
I can give the examples a do/don't heading but they wouldn't change beyond that tbh

there is no equivalent for role, suffix, heading nor variation so there could only be a "don't" section for those. `Chips` simply doesn't support those props that standalone Chip offers :/

overall the goal of this section is to try to provide a minimal amount of guidance of what to do when you try to import `Chip` to use within a `Chips`

<img width="356" alt="image" src="https://github.com/user-attachments/assets/d858c5fc-b7f0-46ac-b858-3a9c956398a0">

the answer of which you should use is "it depends". if you need a `prefix` you must use `Chips/Chip`. 

if you don't, you should probably use the new standalone `Chip` however you should be aware that while it is valid to give `Chip` various props, they will do nothing in the context of `Chips`

it's a bunch of non obvious stuff that we don't have documented anywhere, so that's what this is.